package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What prompted this change?

Reply:
rules_js 1.17.0 now assumes there is a package.json next to the pnpm lock file. There should be in most cases but I think we may need to add an opt-out of that behaviour in a follow-up.
cc @kormide

Reply:
I'd think that should be optional.